src/components/CharacterAI.cpp

```diff
@@ -76,6 +86,7 @@ namespace REGoth
 
     mTurnDirection = TurnDirection::Left;
 
+    // return mVisual->tryPlayTransitionAnimationTo("T_RUNTURNL");
```

Comment:
Did you try to run the turning animations but it doesn't work? Is this code supposed to stay commented out or even stay at all?

Reply:
The code is correct, but there is a bug with the animation importer which makes turning animations look weird. I can remove those lines from the PR. 
I agree that commented out code without any note as to *why* it was commented out isn't great.